python3/templates/debian

```diff
@@ -1,30 +1,46 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright (c) 2005-2007 XenSource, Inc
 
 # Code ripped out of 'xgt' script for now
 from __future__ import print_function
-import commands, xmlrpclib, os, sys, httplib, socket, urllib2, signal
+
+import os
+import signal
+import socket
+import sys
+
+import commands
+import httplib
+import urllib2
+import xmlrpclib
 
 verbose = True
 
+
 ##### begin hack.  Provide xmlrpc over UNIX domain socket (cut+pasted from eliloader):
 class UDSHTTPConnection(httplib.HTTPConnection):
-    """ Stupid hacked up HTTPConnection subclass to allow HTTP over Unix domain
-    sockets. """
+    """Stupid hacked up HTTPConnection subclass to allow HTTP over Unix domain
+    sockets."""
```

Comment:
This is just a copy of a class that is also in `XenAPI.py`, and in the comment in `XenAPI.py`,
there is no reference to being a stupid hack. In fact, this subclass is needed for this kind of modification in order to make local XenAPI calls over Unix Domain sockets possible at all.
```suggestion
    """Subclass of HTTPConnection to HTTP over Unix domain sockets for local XenAPI"""
```
PS: After further looking into it (see further remarks below), I'd like to ask developers if this file is still used: I doubt it, because it is installed as `/opt/xensource/packages/post-install-scripts/debian-etch`, which is Debian Etch 4.0 of 2007. 

I believe the same might be the case for the other template in this directory, called "debug" that is just this:
```bash
#!/bin/sh
# Script should be passed a session_id, VM reference and set of block
# devices via the environment

set > /tmp/debug-install-script
```